src/rpc/ngHandlers/Subscribe.h

```diff
@@ -71,11 +71,42 @@ class BaseSubscribeHandler
     RpcSpecConstRef
     spec() const
     {
+        static auto const booksValidator =
+            validation::CustomValidator{[](boost::json::value const& value, std::string_view key) -> MaybeError {
+                if (!value.is_array())
+                    return Error{RPC::Status{RPC::RippledError::rpcINVALID_PARAMS, std::string(key) + "NotArray"}};
+                for (auto const& book : value.as_array())
+                {
+                    if (!book.is_object())
+                        return Error{
+                            RPC::Status{RPC::RippledError::rpcINVALID_PARAMS, std::string(key) + "ItemNotObject"}};
+                    if (book.as_object().contains("both") && !book.as_object().at("both").is_bool())
+                    {
```

Comment:
May as well remove the extra scope here same as on the line above as there is no complexity to the statement. same for below if statements :)